### DISCUSSION TITLE
Custom Themes and Lay-outs Showcase

### DISCUSSION THREAD

**Discussion by SecularSteve** [searxng/searxng#3071] [Show and tell]
I know there's probably a few of you out there who've gone beyond the Simple, or even Oscar legacy themes.

I for one would like to see what you came up with. That includes sharing your own planned concepts and/or discussing your current designs here.

Keep in mind, this isn't official.

**Comment by vojkovic** [comment]
I added [catppuccin](https://github.com/catppuccin/userstyles/tree/main/styles/searxng) themes to [priv.au](https://priv.au).

You can probably find more on [userstyles.world](https://userstyles.world/).

![image](https://github.com/searxng/searxng/assets/86880643/8280a10d-8867-4bf4-906c-f5d65d768b45)
